src/crimson/osd/pg.cc

```diff
@@ -934,11 +934,7 @@ PG::submit_transaction(
     std::move(submitted),
     all_completed.then_interruptible(
       [this, last_complete=peering_state.get_info().last_complete, at_version]
-      (auto acked) {
-      for (const auto& peer : acked) {
-        peering_state.update_peer_last_complete_ondisk(
-          peer.shard, peer.last_complete_ondisk);
-      }
+      (auto) {
```

Comment:
nit: should we log `acked` peers?

Reply:
Yeah, will do.

Reply:
Thanks, please ping so I'll re label this as needs-qa